### ISSUE TITLE
java.lang.IllegalArgumentException: Component can't be null

### ISSUE THREAD

**Issue by CveniEs** [Backelite/sonar-swift#202]
The SonarQube is 7.3，sonar-swift is 0.4.3，but when I run the run-sonar-swift.sh, can't work well , anyone know how to solve this problem
```
19:38:56.110 ERROR: Error during SonarQube Scanner execution
java.lang.IllegalArgumentException: Component can't be null
	at org.sonar.api.internal.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
	at org.sonar.api.batch.sensor.issue.internal.DefaultIssueLocation.on(DefaultIssueLocation.java:44)
	at com.backelite.sonarqube.objectivec.issues.oclint.OCLintParser.collectFileViolations(OCLintParser.java:98)
	at com.backelite.sonarqube.objectivec.issues.oclint.OCLintParser.parseFiles(OCLintParser.java:81)
	at com.backelite.sonarqube.objectivec.issues.oclint.OCLintParser.parseReport(OCLintParser.java:60)
	at com.backelite.sonarqube.objectivec.issues.oclint.OCLintSensor.execute(OCLintSensor.java:68)
	at org.sonar.scanner.sensor.SensorWrapper.analyse(SensorWrapper.java:45)
	at org.sonar.scanner.phases.SensorsExecutor.execute(SensorsExecutor.java:88)
	at org.sonar.scanner.phases.SensorsExecutor.execute(SensorsExecutor.java:62)
	at org.sonar.scanner.phases.AbstractPhaseExecutor.execute(AbstractPhaseExecutor.java:74)
	at org.sonar.scanner.scan.ModuleScanContainer.doAfterStart(ModuleScanContainer.java:166)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:136)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:122)
	at org.sonar.scanner.scan.ProjectScanContainer.scan(ProjectScanContainer.java:300)
	at org.sonar.scanner.scan.ProjectScanContainer.scanRecursively(ProjectScanContainer.java:295)
	at org.sonar.scanner.scan.ProjectScanContainer.doAfterStart(ProjectScanContainer.java:269)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:136)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:122)
	at org.sonar.scanner.task.ScanTask.execute(ScanTask.java:48)
	at org.sonar.scanner.task.TaskContainer.doAfterStart(TaskContainer.java:82)
	at org.sonar.core.platform.ComponentContainer.startComponents(ComponentContainer.java:136)
	at org.sonar.core.platform.ComponentContainer.execute(ComponentContainer.java:122)
	at org.sonar.scanner.bootstrap.GlobalContainer.executeTask(GlobalContainer.java:131)
	at org.sonar.batch.bootstrapper.Batch.doExecuteTask(Batch.java:116)
	at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:71)
	at org.sonarsource.scanner.api.internal.batch.BatchIsolatedLauncher.execute(BatchIsolatedLauncher.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.sonarsource.scanner.api.internal.IsolatedLauncherProxy.invoke(IsolatedLauncherProxy.java:60)
	at com.sun.proxy.$Proxy0.execute(Unknown Source)
	at org.sonarsource.scanner.api.EmbeddedScanner.doExecute(EmbeddedScanner.java:185)
	at org.sonarsource.scanner.api.EmbeddedScanner.execute(EmbeddedScanner.java:137)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:111)
	at org.sonarsource.scanner.cli.Main.execute(Main.java:75)
	at org.sonarsource.scanner.cli.Main.main(Main.java:61)
```

**Comment by edopelawi** [comment]
Bump. Anyone have same problem with this?

**Comment by zippy1978** [comment]
Fixed in 0.4.4
